README.md

```diff
@@ -3,40 +3,49 @@
 [![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/xlab-uiuc/linux-mcdc/llvm-trunk.yml?label=LLVM%20trunk)](https://github.com/xlab-uiuc/linux-mcdc/actions/workflows/llvm-trunk.yml)
 [![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/xlab-uiuc/linux-mcdc/llvm-18.yml?label=LLVM%2018)](https://github.com/xlab-uiuc/linux-mcdc/actions/workflows/llvm-18.yml)
 
+This project demonstrates MC/DC coverage on the 5.15.y Linux kernel source tree
+using [llvm-cov](https://llvm.org/docs/CommandGuide/llvm-cov.html). Primary
+development of the kernel patch set is being performed in the [xlab-uiuc/llvm-cov](https://github.com/xlab-uiuc/linux-cov)
```

Comment:
Note `xlab-uiuc/llvm-cov` is not public yet. UIUC side would love to open it but we need Boeing's clearance first @svleest